backend/src/main/java/com/ody/meeting/service/MeetingService.java

```diff
@@ -34,37 +32,42 @@
 @Transactional(readOnly = true)
 public class MeetingService {
 
-    private static final String DEFAULT_INVITE_CODE = "초대코드";
-
     private final MateService mateService;
     private final MeetingRepository meetingRepository;
     private final MateRepository mateRepository;
     private final NotificationService notificationService;
 
     @Transactional
     public MeetingSaveResponseV1 saveV1(MeetingSaveRequestV1 meetingSaveRequestV1) {
-        Meeting meeting = meetingRepository.save(meetingSaveRequestV1.toMeeting(DEFAULT_INVITE_CODE));
-        String encodedInviteCode = InviteCodeGenerator.encode(meeting.getId());
-        meeting.updateInviteCode(encodedInviteCode);
+        String inviteCode = generateUniqueInviteCode();
+        Meeting meeting = meetingRepository.save(meetingSaveRequestV1.toMeeting(inviteCode));
         return MeetingSaveResponseV1.from(meeting);
     }
 
+    private String generateUniqueInviteCode() {
+        String inviteCode = InviteCodeGenerator.generate();
```

Comment:
[제안]

do while 문으로 생성 로직을 하나 아껴보는 건 어떤가요?

```suggestion
       do{
          String inviteCode = InviteCodeGenerator.generate()
       }while(meetingRepository.exsitsByInviteCode(inviteCode))
       return inviteCode;
```

Reply:
```
        String inviteCode;
        do {
            inviteCode = InviteCodeGenerator.generate();
        } while (meetingRepository.existsByInviteCode(inviteCode));
        return inviteCode;
```
 콜리가 제시한 방법도 좋지만 do while문을 사용하면 위와 같이 초기화되지 않은 변수를 할당한 채로 반복문을 실행해야해서
while 문을 선택했습니다 !
생성된 초대코드가 이미 존재하면 while문을 안타서 생성 로직수는 똑같아요 !

Reply:
`초기화되지 않은 변수를 할당` 이 포인트를 잊었네요! 답변 감사해요 카키!